packages/sync-actions/src/channels-actions.js

```diff
@@ -0,0 +1,20 @@
+import { buildBaseAttributesActions } from './utils/common-actions'
+
+export const baseActionsList = [
+  { action: 'changeKey', key: 'key' },
+  { action: 'changeName', key: 'name' },
+  { action: 'changeDescription', key: 'description' },
+  { action: 'setAddress', key: 'address' },
+  { action: 'setGeoLocation', key: 'geoLocation' },
+  { action: 'setRoles', key: 'roles' },
```

Comment:
This is simple enough.
Maybe update the sync-actions doc highlighting that we do not support `addRoles` and `removeRoles` for now?

Reply:
I don't see why. They're always a `Array<String>` so there is very little benefit in having add/remove for now. To me it's behind the API and we can iterate on that if customers have 10k channel lists.